.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,29 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or the definition README at
```

Comment:
This is is a dev container for this project that plugins into a the containers functionality in VS code: https://code.visualstudio.com/docs/remote/containers